framework/src/test/java/com/epam/ta/page/CloudPricingCalculatorPage.java

```diff
@@ -0,0 +1,182 @@
+package com.epam.ta.page;
+
+import com.epam.ta.model.Engine;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class CloudPricingCalculatorPage extends AbstractPage {
+
+    @FindBy(xpath = "//*[@id='cloud-site']//iframe")
+    private WebElement iframe;
+
+    @FindBy(id = "myFrame")
+    private WebElement myFrame;
```

Comment:
iframe, myFrame aren't good names. I don't understand for what they need according to naming

Reply:
I will correct the name of the variables.